packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -897,7 +893,7 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
 
     var totalVisibleRowsAboveNode = 0;
     while (!scanNode.isRoot) {
-      final rowsAbove = parent.children.indexWhere((node) {
+      final int rowsAbove = parent.children.indexWhere((node) {
```

Comment:
delete this whole method `selectionRowNumber`. It is not used anywhere.

Reply:
done